README.md

```diff
@@ -1,5 +1,16 @@
 # Substrate SSVM Node
 
+The primary objective of this project was to demonstrate how to establish a substrate node that binding SSVM by EVMC.
```

Comment:
Our objective should be `to bring EWASM runtime from Ethereum community to Substrate to extend substrate ecosystem`.
You mentioned here is the actual work we've done.

Reply:
Replace done.